novawallet/Modules/BrowserNavigation/BrowserNavigationFactory.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+import Operation_iOS
+import UIKit
+
+enum BrowserNavigationFactory {
+    static func createNavigation() -> BrowserNavigationProtocol? {
+        guard let mainAppContainer = findMainContainer() else { return nil }
```

Comment:
I would suggest to check SelectedWalletSettings.shared.value and don't create navigation module if there is no wallet